prime-angular-frontend/src/app/modules/adjudication/pages/site-overview/site-overview.component.ts

```diff
@@ -79,20 +84,40 @@ export class SiteOverviewComponent extends SiteRegistrationContainerComponent im
     }
   }
 
+  public onApproveOrgClaim() {
+    this.busy = this.organizationResource
+      .approveOrganizationClaim(this.orgClaim.organizationId, this.orgClaim.id)
+      .pipe(
+        exhaustMap(() => this.organizationResource.getOrganizationById(this.organization.id))
+      )
+      .subscribe((organization: Organization) => {
+        this.refresh.next(true);
+        this.organization = organization;
+      });
+  }
+
   public ngOnInit(): void {
     super.ngOnInit();
 
     this.createFormInstance();
 
     const { oid, sid } = this.route.snapshot.params;
 
-    this.busy = forkJoin({
-      organization: this.organizationResource.getOrganizationById(oid),
-      site: this.siteResource.getSiteById(sid)
-    }).subscribe(({ organization, site }) => {
-      this.organization = organization;
-      this.site = site;
-      this.form.get('pec').setValue(site.pec);
+    this.busy = forkJoin([
+      this.organizationResource.getOrganizationById(oid),
+      this.siteResource.getSiteById(sid),
+      this.organizationResource.getOrganizationClaimByOrgId(oid)
```

Comment:
HasClaim is already on the organization model returned on line 107, you don't need this call.

Reply:
We still need the claim details